services-camel/src/main/java/edu/unc/lib/cdr/routing/MetaServicesRouter.java

```diff
@@ -41,26 +41,40 @@ public void configure() throws Exception {
         from("{{fcrepo.stream}}")
             .routeId("CdrMetaServicesRouter")
             .to("direct-vm:index.start")
-            .to("direct:process.enhancement");
+            .to("direct:process.ingest");
 
-        from("direct:process.enhancement")
-            .routeId("ProcessEnhancement")
+        from("direct:process.ingest")
+            .routeId("ProcessIngest")
             .filter(simple("${headers[org.fcrepo.jms.eventType]} contains 'ResourceCreation'"
-                    + " && ${headers[org.fcrepo.jms.identifier]} regex '.*original_file'"
+                    + " && ${headers[org.fcrepo.jms.identifier]} regex '.*(original_file|techmd_fits)'"
                     + " && ${headers[org.fcrepo.jms.resourceType]} contains '" + Binary.getURI() + "'"))
+
                 // Trigger binary processing after an asynchronously
                 .threads(enhancementThreads, enhancementThreads, "CdrEnhancementThread")
                 .delay(simple("{{cdr.enhancement.postIndexingDelay}}"))
-                .to("direct:process.binary.original");
+                .to("direct:process.creation");
 
-        from("direct:process.binary.original")
-            .routeId("ProcessOriginalBinary")
+        from("direct:process.creation")
+            .routeId("ProcessCreationEvent")
             .log(LoggingLevel.DEBUG, "Processing binary metadata for ${headers[org.fcrepo.jms.identifier]}")
             .removeHeaders("CamelHttp*")
             .to("fcrepo:{{fcrepo.baseUrl}}?preferInclude=ServerManaged&accept=text/turtle")
             .process(mdProcessor)
+            .choice()
+                .when(simple("${headers[org.fcrepo.jms.identifier]} regex '.*original_file'"))
+                    .to("direct-vm:replication")
+                    .log(LoggingLevel.INFO, "Replication completed for ${headers[org.fcrepo.jms.identifier]}")
```

Comment:
I moved the logging statement into the ReplicationRouter as follows
`from("direct-vm:replication")
            .routeId("CdrReplicationRoute")
            .log(LoggingLevel.INFO, "Calling replication route for ${headers[org.fcrepo.jms.identifier]}")
            .log(LoggingLevel.INFO, "Replication completed for ${headers[org.fcrepo.jms.identifier]}")
            .to("direct:file.replication");`

Not sure whether this is where you wanted it or not.